### PR TITLE
@kanaabe => fixes Desktop Modal QA items

### PR DESCRIPTION
**Pull request by l2succes**
UI
- Terms of use "of" should not be capitalized (luc)
- Change blur to 50px w/ no background color (luc)

Functionality
- Server errors should clear after retrying (luc)
- Enable submit button on forms always, and show errors if they try to submit (luc)
- /log_in and /sign_up are flashing the modal once and going away instead of sticking (luc)

### CODE REVIEW

File: src/Components/Authentication/Desktop/ModalManager.tsx [hunk 1]
```diff
@@ -86,11 +88,11 @@ export class ModalManager extends Component<
           handleSubmit={handleSubmit}
           onFacebookLogin={() =>
             (window.location.href =
-              submitUrls.facebook + "?redirect-to=" + redirectTo)
+              submitUrls.facebook + "?redirect-to=" + options.redirectTo)
```
Comment (kanaabe) [inline src/Components/Authentication/Desktop/ModalManager.tsx]: No action necessary now, but note that we'll need to add more params for analytics later on --

File: src/Components/Authentication/Mobile/SignUpForm.tsx [hunk 2]
```diff
@@ -137,6 +137,8 @@ export const MobileSignUpForm: FormComponentType = props => {
               <Footer
                 mode="signup"
                 handleTypeChange={type => (window.location.href = "/" + type)}
+                onFacebookLogin={props.onFacebookLogin}
+                onTwitterLogin={props.onTwitterLogin}
```
Comment (kanaabe) [inline src/Components/Authentication/Mobile/SignUpForm.tsx]: Thanks for these! 👍

**Comment by kanaabe** [comment]
Nice! Thanks for those tests too. Merge on green

File: src/Components/Authentication/Desktop/ModalManager.tsx (hunk 1 above)
Comment (l2succes) [reply src/Components/Authentication/Desktop/ModalManager.tsx]: 👌🏾 yup

**Pull request merged**